[Tiny PR] Add caution for OL in Astro alerts (#2618)
astronomer/docs: astro/set-up-data-lineage.md

To confirm that your setup is successful, run a Spark job after you save your configuration. After you run this model, click **Lineage** in the Cloud UI and then click **Runs** in the left menu. Your recent Spark job run appears in the table of most recent runs.

## Enable/Disable OpenLineage

> In Astro, OpenLineage is used to extract lineage metadata for your Airflow operators and to configure Astro Alerts. By default, OpenLineage is enabled for your Astro Deployment. 

Reviewer: ```suggestion
Astro uses OpenLineage to extract lineage metadata for your Airflow operators and to send Astro alerts. By default, OpenLineage is enabled for all Astro Deployments. 
```

> To disable it, add the following line to your `Dockerfile`:

Reviewer: Why not as an Astro environment variable?

Author: @jwitz this is also setting an environment variable. If you mean using the `.env` file, I don't prefer it because of two reasons:
- It is only applicable to local, and then you have to remember to set it in the Cloud UI
- you don't have all your settings saved in the source control - which not great for long-term maintenance. 

If we keep everything in the `Dockerfile`, then it gets version controlled and also no separate copies need to be maintained. 

I have added a tip for the alternate setup for now.

`ENV OPENLINEAGE_DISABLED = True`

> Then, restart your Astro project for the changes to take effect.

Reviewer: ```suggestion
Deploy your Astro project for the changes to take effect on your Deployment.
```

> To enable OpenLineage, you can either set `ENV OPENLINEAGE_DISABLED = False` or remove this line from your `Dockerfile`.

Reviewer: ```suggestion
To reenable OpenLineage, you can either set `ENV OPENLINEAGE_DISABLED = False` or remove this line from your `Dockerfile`.
```

> 

Reviewer: ```suggestion
```
